test/application_system_test_case.rb

```diff
@@ -6,10 +6,4 @@ class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
 
   # TODO: remove once https://github.com/rails/rails/pull/47117 is released
   Selenium::WebDriver.logger.ignore(:capabilities)
-
-  Capybara.register_driver :fake_safari do |app|
-    Capybara::RackTest::Driver.new(app,
```

Comment:
Getting rid of this driver since rack test doesn't support webauthn browser auth. I can't use a Safari driver as it requires people to enable automated code execution in their Safari setttings. Thus, I'm just stubbing the method call to `safari?` to return true.